pom.xml

```diff
@@ -58,6 +58,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.]]>
 		<url>https://github.com/mcneilco/acas-roo-server.git</url>
 	</scm>
 	<repositories>
+		<repository>
```

Comment:
Could this `<repository>` be moved into the `jchem` build profile so it isn't contacted during non-jchem builds?